src/codemagic/cli/cli_app.py

```diff
@@ -377,10 +379,13 @@ def execute(self, command_args: Sequence[CommandArg],
         ).execute(**execute_kwargs)
 
 
+_Fn = TypeVar("_Fn", bound=Callable[..., object])
```

Comment:
```suggestion
_Fn = TypeVar('_Fn', bound=Callable[..., object])
```

Flake8 code style checks expect single quotes.